monitoring/variables.tf

```diff
@@ -33,4 +33,10 @@ variable "project_name" {
 variable "tags" {
   description = "Tags to be applied to resources"
   type        = "map"
-}
\ No newline at end of file
+}
+
+variable "lambda_runtime" {
```

Comment:
Perhaps this should be namespaces? "delius_core_monitoring_ lambda_runtime", as this might be needed elsewhere - unless we update all to use same?

Reply:
or we could have a map

Reply:
Looks good to me... afaik, each component has it's own lambda and variables so I'm not sure it matters greatly whether it's `lambda_runtime` or `delius_core_monitoring_lambda_runtime`.  Conversely, if delius-core ever creates another nodejs lambda for something else, then perhaps `delius_core_lambda_runtime` is a more apt name